hoot-core/src/main/cpp/hoot/core/algorithms/WayMatchStringMerger.cpp

```diff
@@ -45,9 +45,9 @@ QString WayMatchStringMerger::SublineMapping::toString() const
   return QString("{start: %1, end: %2, newWay1: %3, way2: %4, subline2: %5, newWay2: %6}")
     .arg(hoot::toString(_start))
     .arg(hoot::toString(_end))
-    .arg(hoot::toString(newWay1 ? newWay1->getElementId() : ElementId()))
-    .arg(hoot::toString(way2 ? way2->getElementId() : ElementId()))
-    .arg(hoot::toString(subline2))
+    .arg(hoot::toString(_newWay1 ? _newWay1->getElementId() : ElementId()))
+    .arg(hoot::toString(_way2 ? _way2->getElementId() : ElementId()))
+    .arg(hoot::toString(_subline2))
```

Comment:
These variables were `public` so when I moved them to `private` I added the preceeding `_`